spec/index.html

```diff
@@ -253,12 +253,15 @@ <h3>RDF Literals</h3>
 
       <p>The corresponding <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">RDF lexical form</a>
         is the characters between the delimiters, after processing any escape sequences.
-        If present, the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
-        is preceded by a '<code>@</code>' (<span class="codepoint">U+0040</span>).
-        If present, the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>
-        is included in the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
-        separated by '<code>--</code>'.
-        If there is no language tag, there may be a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
+        If present, the <a href="#grammar-production-LANGDIR" class="type langTag">LANGDIR</a>
+        terminal matches the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
+        and optionally the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>.
+        The <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
+        is preceded by a '<code>@</code>' (<span class="codepoint">U+0040</span>),
+        and, if present, the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>
+        is separated from the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> by '<code>--</code>'.
+        If there is no matched <a href="#grammar-production-LANGDIR" class="type langTag">LANGDIR</a> terminal,
+        there may be a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
         preceded by '<code>^^</code>' (<span class="codepoint">U+005E</span> <span class="codepoint">U+005E</span>).
```

Comment:
I think there should not be a space between the Unicode codepoints, just as there is no space between the carets... but there may be an argument otherwise
```suggestion
        preceded by two carets ('<code>^^</code>', 
        <span class="codepoint">U+005E</span><span class="codepoint">U+005E</span>).
```

Reply:
I had to look it up and found that Uncode names things as:

U+2038 ‸ [CARET](https://en.wikipedia.org/wiki/Caret_(proofreading))
is not 
U+005E ^ [CIRCUMFLEX ACCENT](https://en.wikipedia.org/wiki/Circumflex)

I suggest keeping it as just `^^` because of this.